modules/build/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -168,6 +169,55 @@ final case class BuildOptions(
       }
     }
 
+  def latestSupportedScalaVersion(prefixScalaVersion: Option[String]): coursier.core.Version = {
+
+    // used when downloading is fail
+    lazy val defaultStableScalaVersions = StableScalaVersion(
+      version,
+      Seq(defaultScala212Version, defaultScala213Version, defaultScalaVersion)
+    )
```

Comment:
This could be moved outside of `latestSupportedScalaVersion` (and be made a `def`, as it's cheap to compute and isn't used 100s of times).